src/App.js

```diff
@@ -227,16 +227,16 @@ class OldApp extends Component {
       <Portal>
         <View style={containerStyle}>
           {this.state.initialRoute && (
-            <RemotePromoSheetProvider isWalletReady={this.props.walletReady}>
-              <RemoteCardProvider>
-                <InitialRouteContext.Provider value={this.state.initialRoute}>
-                  <RoutesComponent onReady={this.handleSentryNavigationIntegration} ref={this.handleNavigatorRef} />
-                  <PortalConsumer />
-                </InitialRouteContext.Provider>
-              </RemoteCardProvider>
-            </RemotePromoSheetProvider>
+            <InitialRouteContext.Provider value={this.state.initialRoute}>
+              <RoutesComponent onReady={this.handleSentryNavigationIntegration} ref={this.handleNavigatorRef} />
+              <PortalConsumer />
+            </InitialRouteContext.Provider>
           )}
           <OfflineToast />
+
+          {/* NOTE: Components below render null and only keep state in sync */}
+          <RemoteCardsSync />
```

Comment:
@christianbaroni we could move the user asset sync component here too if you want